extension/js/common/platform/xss.ts

```diff
@@ -37,6 +37,15 @@ export class Xss {
     return $(selector as any).replaceWith(Xss.htmlSanitize(dirtyHtml)); // xss-sanitized
   }
 
+  public static htmlEscape = (str: string) => {
```

Comment:
We already have a method like this - please use the Xss.escape method below

Reply:
Thanks, how could I have missed that! Fixed.